plugins/Login/SystemSettings.php

```diff
@@ -52,7 +53,9 @@ private function createEnablePasswordStrengthCheck(): Setting
     {
         return $this->makeSetting('enablePasswordStrengthCheck', $default = false, FieldConfig::TYPE_BOOL, function (FieldConfig $field) {
             $field->title = Piwik::translate('Login_SettingPasswordStrengthCheck');
-            $field->description = Piwik::translate('Login_SettingPasswordStrengthCheckHelp');
+            $PasswordStrengthChecker = new PasswordStrength($featureEnabled = true);
+            $rules = $PasswordStrengthChecker->getRules();
+            $field->inlineHelp = Piwik::translate('Login_SettingPasswordStrengthCheckHelp', [$PasswordStrengthChecker->formatRulesListHTML($rules)]);
```

Comment:
Since we don't have a helper that would turn a list into an HTML list, I would probably do something like this so that we don't have to get the rules first and pass them back to the class for formatting when the class already has them.

Note the new/renamed method `getRulesAsHtmlList`.

```suggestion
            $field->inlineHelp = Piwik::translate('Login_SettingPasswordStrengthCheckHelp', [$PasswordStrengthChecker->getRulesAsHtmlList($rules)]);
```